app/Http/Controllers/Admin/News/NewsController.php

```diff
@@ -0,0 +1,120 @@
+<?php
+
+namespace App\Http\Controllers\Admin\News;
+
+use App\Common\Constant;
+use App\Http\Controllers\Admin\Controller;
+use App\Http\Requests\NewsRequest;
+use App\Http\Services\UploadImage\UploadImageService;
+use App\Models\Admin\Category\Category;
+use App\Models\Admin\News\News;
+use Illuminate\Contracts\Foundation\Application;
+use Illuminate\Contracts\View\Factory;
+use Illuminate\Contracts\View\View;
+use Illuminate\Http\RedirectResponse;
+use Illuminate\Http\Request;
+use Illuminate\Routing\Redirector;
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Redirect;
+use Illuminate\Support\Facades\Storage;
+
+class NewsController extends Controller
+{
+    /**
+     * @var News
+     */
+    protected News $news;
+
+    /**
+     * @var UploadImageService
+     */
+    protected UploadImageService $uploadImageService;
+
+    /**
+     * NewsController constructor.
+     *
+     * @param News $news
+     * @param UploadImageService $uploadImageService
+     */
+    public function __construct(News $news, UploadImageService $uploadImageService)
+    {
+        $this->middleware('auth');
+        $this->news = $news;
+        $this->uploadImageService = $uploadImageService;
+    }
+
+    public function index()
+    {
+        $listNews = $this->news->getAll();
+
+        return view(Constant::FOLDER_URL_ADMIN . '.news.index', compact('listNews'));
+    }
+
+    /**
+     * @param Request $request
+     *
+     * @return Application|Factory|View|RedirectResponse
+     */
+    public function createAndEdit(Request $request)
+    {
+        $id = (int)$request->id;
+        $listNews = $this->news->getAll();
```

Comment:
$listNews = $this->news->getAll(); anh không thấy em dùng phần này, em remove đi